### PR TITLE
Feature nav

### PR DESCRIPTION
**Pull request by TraNequaFauntleroy**
I added and styled the collapsible sidebar nav. 

THIS IS STILL A WIP. Video will be uploaded upon completion of feature.

### CODE REVIEW

**Review by stevenjhomem — APPROVED** [review]
Great! We really needed a styled nav bar :)

File: src/styles/ParentStyles/index.less [hunk 1]
```diff
@@ -23,7 +23,6 @@
         h2{
             font-size: 1.5rem;
             color: #fff;
-
```
Comment (zach-morris-txt) [inline src/styles/ParentStyles/index.less:26]: You write and maintain clean code, this is a great example

**Review by zach-morris-txt — CHANGES_REQUESTED** [review]
You have an eye for aestheticism and organization. I looked through the Sider you added. That piece cleaned up the mess that was the Menu/keys/default keys/etc. It's much better for it. Your commit messages are also fantastic. 

This was an accidental PR from what I understand -you have more in mind to complete it and you plan on making a [LOOM] video walkthrough- so I'll leave a "requests changes" review.

File: src/styles/global.less [hunk 2]
```diff
@@ -10,18 +10,21 @@
     color: orange;
 }
 .ant-menu-item-selected {
-    color: #fff !important;
-        background-color: rgba(0, 201, 183, 0.8) !important;
+    color: #fff;
+        background-color: rgba(0, 201, 183, 0.8);
```
Comment (ryan-hamblin) [inline src/styles/global.less:14]: Remove an indent here. Keep your properties on your css Objects aligned.

**Review by ryan-hamblin — COMMENTED** [review]
Keep up the good work, are a few minor formatting issues in your styles. Other than that keep working out this feature!

**Pull request merged**